kola/tests/ignition/kernel.go

```diff
@@ -0,0 +1,29 @@
+// Copyright The Mantle Authors
+// SPDX-License-Identifier: Apache-2.0
+package ignition
+
+import (
+	"github.com/coreos/go-semver/semver"
+	"github.com/flatcar-linux/mantle/kola/cluster"
+	"github.com/flatcar-linux/mantle/kola/register"
+	"github.com/flatcar-linux/mantle/platform/conf"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "cl.ignition.kargs",
+		Run:         check,
+		ClusterSize: 1,
+		UserData: conf.Ignition(`{
+		  "ignition": {
+		    "version": "3.3.0"
+		  },
+		  "kernelArguments": {
+		    "shouldExist": ["quiet"]
+		  }
+		}`),
+		MinVersion: semver.Version{Major: 3178},
+	})
+}
+
+func check(c cluster.TestCluster) {}
```

Comment:
```suggestion
func check(c cluster.TestCluster) {
	m := c.Machines()[0]

	c.AssertCmdOutputContains(m, "cat /proc/cmdline", " quiet") // assuming space for word separation
}
```

Reply:
Can we add this?

Reply:
Oh yes definitely - I totally missed this suggestion. :)